ruby/alias_manager.rb

```diff
@@ -0,0 +1,51 @@
+# for vowels/consonants - compare letter to v/c index

```

Comment:
My suggestions is not perfect but its always good to break down your code into easily manageable methods.

Reach out if you have any questions and keep up the good work!